actions/lib/action.py

```diff
@@ -139,6 +139,17 @@ def do_method(self, module_path, cls, action, **kwargs):
             zone = kwargs['zone']
             del kwargs['zone']
             obj = self.get_r53zone(zone)
+        elif module_path == 'boto3.s3.transfer':
+           for k, v in kwargs.items():
+                if not v:
+                    del kwargs[k]
```

Comment:
Maybe better skip the remaining operations in the loop? Otherwise you could get `KeyError` exceptions when the key matches, but the value is empty.
```suggestion
                    del kwargs[k]
                    continue
```

Reply:
You're perfectly right , thanks !